src/PasslessServiceProvider.php

```diff
@@ -18,6 +18,8 @@ public function register()
         $this->mergeConfigFrom(
             __DIR__ . '/../config/passless.php', 'passless'
         );
+
+        $this->app->bind(LoginNotification::class, $this->app['config']['passless']['notification']);
```

Comment:
> "If the Service Container is called Service Container, it's because it contains fucking services.". -- Me, 2017.

It's better to set it in the [PasslessGuard](https://github.com/DarkGhostHunter/Passless/blob/master/src/PasslessGuard.php#L61). Did I forgot to add this? Was I too drunk?

```php
// Check first if the config has a notification class set. If there is one, we will tell the Service Container to
// instance it, and the return it so it can be sent. We will inject the Login Notification the "remember"
// and "intended" variables forcefully in the notification so the dev can use them at login time.
$notification = app('config')->get('passless.notification') ?? LoginNotification::class;

$notification = app()->make($notification, [
    'remember' => $remember,
    'intended' => $intended
]);

app(Dispatcher::class)->send($user, $notification);
```